internal/provider/tfmodels/deployment_table.go

```diff
@@ -0,0 +1,33 @@
+package tfmodels
+
+import (
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"terraform-provider-artie/internal/artieclient"
+)
+
+type Table struct {
+	UUID                 types.String `tfsdk:"uuid"`
+	Name                 types.String `tfsdk:"name"`
+	Schema               types.String `tfsdk:"schema"`
+	EnableHistoryMode    types.Bool   `tfsdk:"enable_history_mode"`
+	IndividualDeployment types.Bool   `tfsdk:"individual_deployment"`
+	IsPartitioned        types.Bool   `tfsdk:"is_partitioned"`
+}
+
+func (t Table) ToAPIModel() artieclient.Table {
+	tableUUID := t.UUID.ValueString()
```

Comment:
Could do:
```
var tableUUID uuid.UUID
if t.UUID.ValueString() != "" {
tableUUID = uuid.MustParse(t.UUID.ValueString())
}
```
So that you don't have to parse the zero UUID.

Reply:
Or `tableUUID := uuid.Nil` if you want to be a bit more explicit.